te/tedp_docker/setup.sh

```diff
@@ -29,50 +29,60 @@
 # OF SUCH DAMAGE
 #**********************************************************************************************
 
-#!/bin/sh
-apt update  # To get the latest package lists
+#!/bin/bash
 cd /tmp/
+sed -i '/deb-src/s/^# //' /etc/apt/sources.list && apt update
 apt install automake libtool m4 -y
 apt install libjson0 libjson0-dev -y
 apt build-dep curl -y
 apt install wget -y
-apt install build-essential nghttp2 libnghttp2-dev libssl-dev -y
+apt install build-essential nghttp2 libnghttp2-dev -y
+
+###################### Download all first ######################
```

Comment:
Do the downloads followed by compilation. `wget` is a temporary package that depends on libssl1.0. So post getting the source code, libssl can be uninstalled